src/components/Site/Site.jsx

```diff
@@ -137,6 +127,14 @@ class Site extends React.Component {
    * @return {array}       - ...
    */
   _strip = array => {
+    let anchorTitleIndex = array.findIndex(item => item.name === 'index.md');
```

Comment:
i would suggest going for `item.name.toLowercase()` in case if someone accidentally makes `index.MD` in the future

Reply:
@EugeneHlushko Updated